src/rgw/rgw_sal_motr.cc

```diff
@@ -3297,7 +3297,29 @@ int MotrMultipartUpload::list_parts(const DoutPrefixProvider *dpp, CephContext *
   string upload_id = get_upload_id();
 
   if(upload_id.length() == 0){
-    rc = store->get_upload_id(tenant_bkt_name, this->get_key(), upload_id);
+    std::unique_ptr<rgw::sal::Object> obj_ver = this->bucket->get_object(rgw_obj_key(this->get_key()));
+    rgw::sal::MotrObject *mobj_ver = static_cast<rgw::sal::MotrObject *>(obj_ver.get());
+    rgw_bucket_dir_entry ent;
+    std::string key_name;
+
+    // Get the object entry
+    int ret_rc = mobj_ver->get_bucket_dir_ent(dpp, ent);
+    if(ret_rc < 0)
```

Comment:
log error with rc

Reply:
The error messages are given in the `get_bucket_dir_ent`() function. Hence checking only `ret_rc` in the function.